### ISSUE TITLE
add language tests

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [Abdur-rahmaanJ/greenBerry#51]

**Comment by Abdur-rahmaanJ** [comment]
@georgeoffley still open.

By testing means test syntax input against expected behaviour

**Comment by georgeoffley** [comment]
> @georgeoffley still open.
> 
> By testing means test syntax input against expected behaviour

No, I understand what tests are, I just need a bit more context. I see you there is a gbtest directory. Is that the tests for the actual new language? Do you mean adding Python unit tests as well?

**Comment by Abdur-rahmaanJ** [comment]
Yes tests for the tools yes python unittests, i suggest pytest.

**Comment by Abdur-rahmaanJ** [comment]
@georgeoffley Anything just ping. If you need to modify some parts to make stuffs more clear, just do it

**Comment by georgeoffley** [comment]
> @georgeoffley Anything just ping. If you need to modify some parts to make stuffs more clear, just do it

Understood, thanks. I'll get started then!
